README.org

```diff
@@ -112,9 +112,6 @@ You can also fork this repository and use your own repo.
 **** manual installation
 Clone this repo and make sure the files are on your load path, as described on [[https://www.emacswiki.org/emacs/LoadPath][EmacsWiki]].
 
-*Make sure you load consult (e.g. =require consult=) before you load consult-gh.*
```

Comment:
You should keep this in the README because this is still a good reminder for manual installation!

Reply:
Okay. Just removed that change.